### DISCUSSION TITLE
Slider Input for Extensions

### DISCUSSION THREAD

**Discussion by c00lkase** [TurboWarp/extensions#1399] [Ideas]
Similar to MakeCode, a slider input would be nice just to not have to deal with really high or really low values. Here's an example I made of how it could look:

![value dropdown](https://github.com/TurboWarp/extensions/assets/90591350/ed573151-04ad-423f-9e87-f70e0a3288b4)

**Comment by snowboyz0825** [comment]
This would be more of an add on on thing

**Comment by CubesterYT** [comment]
This would have to be a new type in scratch blocks, so it's impossible to add any other way.
